public/locales/ar/translation.json

```diff
@@ -19,6 +19,13 @@
     }
   },
   "home": {
+    "genderViolenceMain": {
+      "headerPartOne": "!أنت لست لوحدك",
```

Comment:
avoid using partOne and partTwo or header0 and header1 in general

give these two more meaningful names

aloneStatement
safePlaceStatement
or something like this